src/celestia/gtk/menu-context.h

```diff
@@ -21,11 +21,14 @@
 
 #include "common.h"
 
+class GTKContextMenuHandler : public CelestiaCore::ContextMenuHandler {
+public:
```

Comment:
before public/private we use 1 space indent

Reply:
i fixed this one but it seems our win32 code doesn't follow that